src/20241201_Multi_IncreaseBorrowSlope1ToAllStablecoinsAcrossAllAaveInstances/IncreaseBorrowSlope1ToAllStablecoinsAcrossAllAaveInstances_20241201.s.sol

```diff
@@ -0,0 +1,410 @@
+        .creationCode
+    );
+    actionsAvalanche[1] = GovV3Helpers.buildAction(
+      type(AaveV3Avalanche_IncreaseBorrowSlope1ToAllStablecoinsAcrossAllAaveInstances_20241201)
+        .creationCode
+    );
+    payloads[2] = GovV3Helpers.buildAvalanchePayload(vm, actionsAvalanche);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsOptimism = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsOptimism[0] = GovV3Helpers.buildAction(
+      type(AaveV3Optimism_IncreaseBorrowSlope1ToAllStablecoinsAcrossAllAaveInstances_20241201)
+        .creationCode
+    );
+    payloads[3] = GovV3Helpers.buildOptimismPayload(vm, actionsOptimism);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsArbitrum = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsArbitrum[0] = GovV3Helpers.buildAction(
+      type(AaveV3Arbitrum_IncreaseBorrowSlope1ToAllStablecoinsAcrossAllAaveInstances_20241201)
+        .creationCode
+    );
+    payloads[4] = GovV3Helpers.buildArbitrumPayload(vm, actionsArbitrum);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsMetis = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsMetis[0] = GovV3Helpers.buildAction(
+      type(AaveV3Metis_IncreaseBorrowSlope1ToAllStablecoinsAcrossAllAaveInstances_20241201)
+        .creationCode
+    );
+    payloads[5] = GovV3Helpers.buildMetisPayload(vm, actionsMetis);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsBase = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsBase[0] = GovV3Helpers.buildAction(
+      type(AaveV3Base_IncreaseBorrowSlope1ToAllStablecoinsAcrossAllAaveInstances_20241201)
+        .creationCode
+    );
+    payloads[6] = GovV3Helpers.buildBasePayload(vm, actionsBase);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsGnosis = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsGnosis[0] = GovV3Helpers.buildAction(
+      type(AaveV3Gnosis_IncreaseBorrowSlope1ToAllStablecoinsAcrossAllAaveInstances_20241201)
+        .creationCode
+    );
+    payloads[7] = GovV3Helpers.buildGnosisPayload(vm, actionsGnosis);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsScroll = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsScroll[0] = GovV3Helpers.buildAction(
+      type(AaveV3Scroll_IncreaseBorrowSlope1ToAllStablecoinsAcrossAllAaveInstances_20241201)
+        .creationCode
+    );
+    payloads[8] = GovV3Helpers.buildScrollPayload(vm, actionsScroll);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsBNB = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsBNB[0] = GovV3Helpers.buildAction(
+      type(AaveV3BNB_IncreaseBorrowSlope1ToAllStablecoinsAcrossAllAaveInstances_20241201)
+        .creationCode
+    );
+    payloads[9] = GovV3Helpers.buildBNBPayload(vm, actionsBNB);
+
+    IPayloadsControllerCore.ExecutionAction[]
```

Comment:
For zksync the payload should be added manually, once deployed on that network. You can check an example here:
https://github.com/bgd-labs/aave-proposals-v3/pull/466/files#diff-a7c113bcf00f556911a37fc533aab4be25abdd853b43b2f345d260b2fbbfa1d4R268